File: src/main/java/net/kenro/ji/jin/purescript/psi/impl/PSNumericLiteralImpl.java
package net.kenro.ji.jin.purescript.psi.impl;

import com.intellij.lang.ASTNode;
import com.intellij.psi.PsiElementVisitor;
import net.kenro.ji.jin.purescript.psi.PSNumericLiteral;
import net.kenro.ji.jin.purescript.psi.PSVisitor;
import org.jetbrains.annotations.NotNull;

public class PSNumericLiteralImpl extends PSPsiElement implements PSNumericLiteral {

    public PSNumericLiteralImpl(final ASTNode node) {
        super(node);
    }

}

File: src/main/java/net/kenro/ji/jin/purescript/psi/impl/PSNumericLiteralImpl.kt
package net.kenro.ji.jin.purescript.psi.impl

import com.intellij.lang.ASTNode
import net.kenro.ji.jin.purescript.psi.PSNumericLiteral

class PSNumericLiteralImpl(node: ASTNode) : PSPsiElement(node), PSNumericLiteral